python-sdk/tests_integration/databases/test_all_databases.py

```diff
@@ -118,9 +123,10 @@ def test_export_table_to_pandas_dataframe_mssql(
         {"database": Database.SQLITE},
         {"database": Database.DELTA},
         {"database": Database.MSSQL},
+        {"database": Database.DUCKDB},
     ],
     indirect=True,
-    ids=["bigquery", "postgres", "redshift", "snowflake", "sqlite", "delta", "mssql"],
+    ids=["bigquery", "postgres", "redshift", "snowflake", "sqlite", "delta", "mssql", "duckdb"],
```

Comment:
By looking at how many places you had to make this type of change, I was wondering if we could, by any chance, try to compute things using `SUPPORTED_DATABASES`:
https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/constants.py#L70

So we don't need the list hard-coded in so many tests. If we managed to get this to work, it would be much easier to add future databases, since we wouldn't need to update as many tests

Reply:
Maybe this can be helpful as well - https://hackebrot.github.io/pytest-tricks/param_id_func/

Reply:
I agree. We should have a marker that makes the test case run for all databases, without us having to redefine databases or ids.

Reply:
Should this change be part of this PR? I think that can be a separate issue. WDYT?

Reply:
Happy to have it logged as a separate ticket! Please, could you add the link here, @sunank200 , and help us prioritise it?

Reply:
created the ticket here: https://github.com/astronomer/astro-sdk/issues/1711